logstash-core/src/main/java/org/logstash/common/SourceWithMetadata.java

```diff
@@ -39,6 +40,11 @@ public String getText() {
         return text;
     }
 
+    @JRubyMethod(name = "lines_count")
```

Comment:
nitpick: I would avoid pulling jruby into this class, and just use the java method name from jruby.